tests/test_utils/server/litellm_server_config.yaml

```diff
@@ -7,6 +7,7 @@ model_list:
       model: "dspy-test-provider/dspy-test-model"
 
 litellm_settings:
+  num_retries: 0
```

Comment:
Disable retries on the server side to ensure that server retries don't stack atop client retries, which can result in test failures due to a mismatch between expected and actual # of retries